package.json

```diff
@@ -40,14 +40,14 @@
     "sinon": "^7.2.7"
   },
   "dependencies": {
+    "@hapi/hoek": "^9.0.4",
     "@octokit/rest": "~16.33",
     "@octokit/webhooks": "~7.6.2",
     "circuit-fuses": "^4.0.4",
-    "hoek": "^6.1.2",
-    "joi": "^13.7.0",
-    "screwdriver-data-schema": "^19.1.1",
-    "screwdriver-scm-base": "^6.0.0",
-    "screwdriver-logger": "^1.0.0"
+    "joi": "^17.1.1",
+    "screwdriver-data-schema": "git://github.com/screwdriver-cd/data-schema.git#hapi-v19",
```

Comment:
We can wait until https://github.com/screwdriver-cd/datastore-base/pull/37#pullrequestreview-460320174 merged to update version.